### ISSUE TITLE
german umlauts in node name

### ISSUE THREAD

**Issue by mwarning** [ffulm/server-config#20]
If alfred outputs nam fields like "T\xc3\xa4nnesberger_Weg_42", then ffmap-backend.py produces an error. Also, the name field is not limited in length and might be used to spam the map.

**Comment by mwarning** [comment]
I would like to propose to replace
    "name": { "type": "string", "pattern": NAME_RE },
by
    "name": { "type": "string", "maxLength": 30 },

**Comment by mwarning** [comment]
These two line should fix the charset:
line = bytes(line, 'ascii').decode("unicode_escape")
line = line.encode('latin-1').decode('utf8')

**Comment by saraedum** [comment]
I agree that we should limit the length of the names (or maybe rather trim long strings.)
I am not seeing any issues with umlauts: http://vpn1.ulm.freifunk.net/map/list.html (vpn1.äöü)
What kind of client of node is reporting this name?

**Comment by mwarning** [comment]
The problem ist with the graph (http://vpn1.ulm.freifunk.net/map/graph.html).
I have a fix ready. But it is not very nice yet: https://github.com/freifunk-bielefeld/server-config/commit/7022c31867dd8b61252aa4891ead33a7156da0eb

**Comment by saraedum** [comment]
Here, the graph looks just fine, i.e., vpn1.äöü does show up.
